receiver/awsecscontainermetricsreceiver/awsecscontainermetrics/label.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2020, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package awsecscontainermetrics
+
+import (
+	"strings"
+
+	metricspb "github.com/census-instrumentation/opencensus-proto/gen-go/metrics/v1"
+)
+
+func containerLabelKeysAndValues(cm ContainerMetadata) ([]*metricspb.LabelKey, []*metricspb.LabelValue) {
+	labelKeys := make([]*metricspb.LabelKey, 0, 3)
+	labelValues := make([]*metricspb.LabelValue, 0, 3)
+
+	labelKeys = append(labelKeys, &metricspb.LabelKey{Key: "container.name"})
```

Comment:
I've been told using the convention constants directly where applicable is preferred: https://github.com/open-telemetry/opentelemetry-collector/blob/master/translator/conventions/opentelemetry.go

Reply:
Good point. Used the default convention. Also, transferred some of my values to a constant file.